content/_partials/dropdown.html.haml

```diff
@@ -0,0 +1,2 @@
+%button.nav-link.dropdown-toggle{"aria-expanded" => "false", "aria-haspopup" => "true", "data-toggle" => "dropdown"}
```

Comment:
Did this refactoring intentionally drop the `:role => "button"` ?  I don't detect any difference in my side by side comparisons using Firefox and Chrome on my Ubuntu 16 machine, but would like to be sure that is an intentional change.

I think the removal of the `:role => "button"` looks the same in the new as in the old because you've implemented the CSS in jenkins.css to represent that markup.  Have I understood correctly?

Reply:
Yes, that's intentional, `role="button"` was needed for `div` elements but is redundant for `button`.